docs/modules/record-encryption/con-record-encryption-overview.adoc

```diff
@@ -73,7 +73,11 @@ It is recommended to use the KMS in a high availability (HA) configuration.
 Key rotation involves periodically replacing cryptographic keys with new ones and is considered a best practice in cryptography.
 
 The filter allows the rotation of Key Encryption Keys (KEKs) within the Key Management System (KMS). 
-When a KEK is rotated, the new key material is used for newly produced records. Existing records, encrypted with older KEK versions, remain decryptable as long as the previous KEK versions are still in the KMS.
+When a KEK is rotated, the new key material is eventually used for newly produced records. Existing records, encrypted with older KEK versions, remain decryptable as long as the previous KEK versions are still in the KMS.
+
+When the KEK is rotated in the external KMS, it will take up to 2 hours (by default) before all records produced by the filter
```

Comment:
I almost wonder if `2 hours` is too short given the cost implications of that decision. Maybe 6 or 12 would make for a better default.

Reply:
I realised I need to document the refresh config option too, it's more like 1hour effectively. 2hours is when it's eligible for eviction. We can bump the defaults and doc together in another PR if it's too small.

Reply:
I think the defaults are good enough for now.